modules/ROOT/pages/authentication-authorization/manage-roles.adoc

```diff
@@ -573,9 +579,19 @@ This is equivalent to running `DROP ROLE myrole IF EXISTS` followed by `CREATE R
 
 [NOTE]
 ====
-* The `CREATE OR REPLACE ROLE` command does not allow you to use the `IF NOT EXISTS`.
+The `CREATE OR REPLACE ROLE` command does not allow you to use the `IF NOT EXISTS`.
 ====
 
+[role=new-in-5.26]
+[[access-control-immutable-roles]]
+== Immutable roles
+
+Immutable roles are those that cannot be modified in the usual way.
+This means they cannot be created, renamed, dropped, or have privileges granted to or revoked from them.
```

Comment:
This contradicts the above commands, for example, CREATE [IMMUTABLE] ROLE name AS COPY OF otherName or CREATE OR REPLACE [IMMUTABLE] ROLE name [AS COPY OF otherName], etc.